lms/static/sass/shared-v2/_layouts.scss

```diff
@@ -1,57 +1,77 @@
 // LMS layouts
 
 .content-wrapper {
-    max-width: $lms-max-width;
-    padding-bottom: $baseline*2;
+  max-width: $lms-max-width;
+  padding-bottom: $baseline*2;
 
-    .container {
-        @include clearfix();
-        border: 1px solid $lms-border-color;
-        background-color: $lms-container-background-color;
+  .page-content-container {
+    @include clearfix();
+
+    border: 1px solid $lms-border-color;
+    background-color: $lms-container-background-color;
+  }
+
+  .page-header {
+    @include clearfix();
+
+    border-bottom: 1px solid $lms-border-color;
+    padding: $baseline;
+
+    .page-title {
+      @extend %t-title4;
+
+      margin-bottom: ($baseline/4);
+      text-transform: none;
+      color: $black;
     }
 
-    .page-header {
-        @include clearfix();
-        border-bottom: 1px solid $lms-border-color;
-        padding: $baseline;
+    .page-description {
+      @extend %t-copy-sub1;
 
-        .page-title {
-            @extend %t-title4;
-            margin-bottom: ($baseline/4);
-            text-transform: none;
-            color: $black;
-        }
+      margin-bottom: ($baseline/4);
+      color: $gray;
+    }
 
-        .page-description {
-            @extend %t-copy-sub1;
-            margin-bottom: ($baseline/4);
-            color: $gray;
+    &.has-secondary {
+      .page-header-main {
+        display: inline-block;
+      }
+
+      .page-header-secondary {
+        @include float(right);
+        @include text-align(right);
+
+        display: inline-block;
+        vertical-align: text-bottom;
+
+        .form-actions {
+          @include margin-left($baseline/2);
+
+          display: inline-block;
         }
 
-        &.has-secondary {
-            .page-header-main {
-                display: inline-block;
-            }
-
-            .page-header-secondary {
-                @include float(right);
-                @include text-align(right);
-                display: inline-block;
-                vertical-align: text-bottom;
-
-                .form-actions {
-                    @include margin-left($baseline/2);
-                    display: inline-block;
-                }
-
-                .form-actions > *:first-child {
-                    @include margin-left(0);
-                }
-            }
+        .form-actions > *:first-child {
+          @include margin-left(0);
         }
+      }
     }
+  }
+
+  .page-content {
+    padding: $baseline;
+
+    @media (min-width: $grid-breakpoints-md) {
+      display: flex;
+
+      .page-content-main {
+        flex-grow: 1;  // This column should consume all the available space
+      }
 
-    .page-content {
-        padding: $baseline;
+      .page-content-secondary {
+        min-width: 350px;
+        max-width: 350px;
```

Comment:
Why not just ```width: 350px```?

Reply:
Same problem with the width ending up smaller in some instances.

Reply:
I think it would be worth leaving a comment here explaining this, so if someone in the future figures out how to resolve the issue they can update the code with more confidence.

Reply:
Done